src/modules/flow/lsm9ds0/lsm9ds0.json

```diff
@@ -0,0 +1,127 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "lsm9ds0",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "Apache-2.0",
+    "version": "1"
+  },
+  "types": [
+    {
+      "name": "lsm9ds0/gyroscope",
+      "category": "input/hw",
+      "description": "Gyroscope node for the lsm9ds0 chipset.",
+      "methods": {
+        "close": "lsm9ds0_gyro_close",
+        "open": "lsm9ds0_gyro_open"
+      },
+      "options": {
+       "members": [
+         {
+           "name": "i2c_bus",
+           "data_type": "int",
+           "description": "I2C bus number"
+         },
+         {
+           "name": "i2c_addr",
+           "data_type": "int",
+           "description": "I2C address",
+           "default": 107
+         },
+         {
+           "name": "scale",
+           "data_type": "int",
+           "description": "Scale selection (in degrees per second). Must be one of 245, 500 or 2000. The scales ranges from -value to +value",
+           "default": 245
+         }
+       ],
+       "version": 1
+      },
+      "in_ports": [
+        {
+          "name": "TICK",
+          "data_type": "any",
+          "description": "Every time a packet is received in this port the node will read the sensor value and output it's data.",
+          "methods": {
+              "process": "lsm9ds0_gyro_process"
+          }
+        }
+      ],
+      "out_ports": [
+        {
+          "name": "OUT",
+          "data_type": "direction-vector",
+          "description": "Gyroscope in all X/Y/Z axes, in a scale of degrees per second."
+        }
+      ],
+      "private_data_type": "lsm9ds0_gyro_data",
+      "url": "http://solettaproject.org/doc/latest/node_types/lsm9ds0/gyroscope.html"
+    },
+    {
+      "name": "lsm9ds0/xmt",
```

Comment:
are the usages independent? I mean, can I use it only as thermometer? We could then split the init commands into three sets and have 3 nodes, no? This would really match our expected/pristine nodes API.